### PR TITLE
Added a formal group policy document

### LINKED ISSUE #4: Finalize a document on group policy
This should detail concerns about:

- Tardiness
- Include our existing policy file in this
- How code reviews should be done
- Formal processes for pushing to the repository
- Formal processes for creating issues before working on a features
- etc.

### PR DESCRIPTION
**Pull request by braunsonm**
This document outlines some of the ideas that were already discussed
at meetings but not yet formalized. This is part of the required
documents to be submitted with ID1.

This is subject to approval through a peer review.
I specifically added @eileenvanheerde as she mentioned she’s pretty good at editing documents, and the project lead @aaz713 to confirm policy.

This PR will skip the develop branch.

Closes #4

### CODE REVIEW

File: documents/policy/policy.tex [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+
+\begin{table}[h!]
+\centering
+\begin{tabular}{ |p{2cm}||p{8cm}|  }
+    \hline
+    \multicolumn{2}{|c|}{Team Roles} \\
+    \hline
+    Member & Roles\\
+    \hline
+    Amanda & Project Lead\\
```
Comment (braunsonm) [inline documents/policy/policy.tex]: Please review these to ensure all roles are correct. I guessed for a few of theme.

Comment (eileenvanheerde) [reply documents/policy/policy.tex]: Should be:
Evan - Project Lead
Eileen - Test Lead 
Amanda - Developer

File: documents/policy/policy.tex [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+
+\section{Roles and Responsibilities}
+
+\subsection{Project Lead}
+
+The project lead will be responsible for organizing meetings,
+meeting with stakeholders, establishing priority, coordinating
+between accountable positions, and other administrative work. 
+Generally, the project lead will not directly code.
+In cases of disagreements the project lead has executive decision making power.
```
Comment (eileenvanheerde) [inline documents/policy/policy.tex]: Establishing priority of what? Should probably clarify that a bit. Also, I'm assuming "administrative work" includes emailing the stakeholders, taking meeting minutes, etc. (Just wondering how much detail we want with that).

File: documents/policy/policy.tex [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+
+The risk officer is a rotating, part-time role which evaluates
+risk continuously during project development. They are responsible
+for documenting mitigation strategies and proactively taking
+steps to reduce risks to the project. 
+
+\section{Meetings}
+
+Meetings and class room attendance is considered mandatory. 
+If an individual cannot attend a meeting they are to notify the project lead ahead of time and read the meeting minutes after. If more than three meetings are missed in the course of the term that person must talk to Amanda, if any more meetings are missed the team will talk to Dr. Osgood.
```
Comment (eileenvanheerde) [inline documents/policy/policy.tex]:
```suggestion
If an individual cannot attend a meeting they are to notify the project lead ahead of time and must read the meeting minutes afterwards/when they are able to/at their earliest opportunity. If more than three meetings are missed in the course of the term that person must talk to Amanda, if any more meetings are missed, the team will talk to Dr. Osgood.
```

File: documents/policy/policy.tex [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+
+Ten minute meetings before class will be used as an informal stand up.
```
Comment (eileenvanheerde) [inline documents/policy/policy.tex]:
```suggestion
The first ten minutes of class will be used as an informal stand up.
```

File: documents/policy/policy.tex (hunk 2 above)
Comment (braunsonm) [reply documents/policy/policy.tex]: Adjusted the priority line.

And yea I note in this paragraph they are responsible for organizing meetings (agenda/minutes), and I mention meeting with stakeholders. Other administrative work could be emailing them right.

File: documents/policy/policy.tex [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+
+The risk officer is a rotating, part-time role which evaluates
+risk continuously during project development. They are responsible
+for documenting mitigation strategies and proactively taking
+steps to reduce risks to the project. 
+
+\section{Meetings}
+
+Meetings and class room attendance is considered mandatory. 
+If an individual cannot attend a meeting they are to notify the project lead ahead of time and must read the meeting minutes afterwards/when they are able to/at their earliest opportunity. If more than three meetings are missed in the course of the term that person must talk to Amanda, if any more meetings are missed, the team will talk to Dr. Osgood.
```
Comment (eileenvanheerde) [inline documents/policy/policy.tex]:
```suggestion
If an individual cannot attend a meeting they are to notify the project lead ahead of time and must read the meeting minutes afterwards. If more than three meetings are missed in the course of the term that person must talk to Amanda, if any more meetings are missed, the team will talk to Dr. Osgood.
```

**Pull request merged**